src/math/Vector2.js

```diff
@@ -482,6 +482,15 @@ Object.assign( Vector2.prototype, {
 
 		return this;
 
+	},
+	
```

Comment:
Please remove all trailing spaces from your PR.

BTW: If you work with editors like VSCode, this is done automatically by the eslint autocorrect.

Reply:
Hhmm I tried this with the default eslint settings. It does remove the trails but messes up the rest of the document. For now I did it by hand, I will have to figure this out, using the eslint-config-mdcs. It's very nice.